torchaudio/functional/functional.py

```diff
@@ -1631,3 +1632,23 @@ def rnnt_loss(
         return costs.sum()
 
     return costs
+
+
+def apply_beamforming(beamform_weights: Tensor, specgram: Tensor) -> Tensor:
+    r"""Apply the beamforming weight to the noisy multi-channel spectrograms to get the single-channel enhanced spectrogram.
+    .. math::
```

Comment:
The rendering is broken.

https://556260-90321822-gh.circle-artifacts.com/0/docs/functional.html#torchaudio.functional.apply_beamforming

Probably blank lines are required around the directive and the equation.

```

Apply the beamforming weight to the noisy multi-channel spectrograms to get the single-channel enhanced spectrogram.

.. math::

   \hat{\textbf{S}}(f) = \textbf{w}_{\text{bf}}(f)^{\mathsf{H}} \textbf{Y}(f)
```